### PR TITLE
Update web to 1.0.1

### PR DESCRIPTION
**Pull request by kulmann**
Update web to most recent release 1.0.1. This also brings a feature which was already used in the tests, therefor we can remove some expected failures:
Fixes https://github.com/owncloud/ocis/issues/1197

### CODE REVIEW

**Review by LukasHirt — APPROVED** [review]

File: tests/acceptance/expected-failures-webUI-on-OWNCLOUD-storage.txt [hunk 1]
```diff
@@ -189,10 +189,6 @@ webUIPreview/imageMediaViewer.feature:33
 # https://github.com/owncloud/web/issues/4296 media viewer does not display jpeg file
 webUIPreview/imageMediaViewer.feature:15
 #
-#
-webUIPreview/imageMediaViewer.feature:100
-webUIPreview/imageMediaViewer.feature:114
```
Comment (phil-davis) [inline tests/acceptance/expected-failures-webUI-on-OWNCLOUD-storage.txt:194]: Note: I had accidentally duplicated these in the expected-failures list. They also appear at the end, around lines 423, 424.

**Comment by phil-davis** [comment]
Some strange API test failure: https://drone.owncloud.com/owncloud/ocis/2636/19/7
```
  Scenario Outline: Creating a share of a file with a user                                                 # /drone/src/tests/acceptance/features/apiBasic/createShareToSharesFolder.feature:4
    Given the administrator has set the default folder for received shares to "Shares"                     # OccContext::theAdministratorHasSetTheDefaultFolderForReceivedSharesTo()
    And auto-accept shares has been disabled                                                               # FeatureContext::autoAcceptSharesHasBeenDisabled()
    And using OCS API version "<ocs_api_version>"                                                          # FeatureContext::usingOcsApiVersion()
    And user "Alice" has been created with default attributes and without skeleton files                   # FeatureContext::userHasBeenCreatedWithDefaultAttributesAndWithoutSkeletonFiles()
    And user "Alice" has uploaded file with content "ownCloud test text file 0" to "/textfile0.txt"        # FeatureContext::userHasUploadedAFileWithContentTo()
    And user "Brian" has been created with default attributes and without skeleton files                   # FeatureContext::userHasBeenCreatedWithDefaultAttributesAndWithoutSkeletonFiles()
    When user "Alice" shares file "textfile0.txt" with user "Brian" using the sharing API                  # FeatureContext::userSharesFileWithUserUsingTheSharingApi()
    And user "Brian" accepts share "/textfile0.txt" offered by user "Alice" using the sharing API          # FeatureContext::userReactsToShareOfferedBy()
    Then the HTTP status code should be "200"                                                              # FeatureContext::thenTheHTTPStatusCodeShouldBe()
    And the content of file "/Shares/textfile0.txt" for user "Brian" should be "ownCloud test text file 0" # FeatureContext::contentOfFileForUserShouldBe()

    Examples:
      | ocs_api_version |
      | 1               |
        Sharing::userReactsToShareOfferedBy could not find share /textfile0.txt, offered by Alice to Brian
        Failed asserting that null is not null.
      | 2               |
        Sharing::userReactsToShareOfferedBy could not find share /textfile0.txt, offered by Alice to Brian
        Failed asserting that null is not null.
```

How is that related to the changes in this PR???

The PR needs a rebase anyway. Maybe that will help.

**Comment by kulmann** [comment]
> Some strange API test failure: https://drone.owncloud.com/owncloud/ocis/2636/19/7
> 
> ```
>   Scenario Outline: Creating a share of a file with a user                                                 # /drone/src/tests/acceptance/features/apiBasic/createShareToSharesFolder.feature:4
>     Given the administrator has set the default folder for received shares to "Shares"                     # OccContext::theAdministratorHasSetTheDefaultFolderForReceivedSharesTo()
>     And auto-accept shares has been disabled                                                               # FeatureContext::autoAcceptSharesHasBeenDisabled()
>     And using OCS API version "<ocs_api_version>"                                                          # FeatureContext::usingOcsApiVersion()
>     And user "Alice" has been created with default attributes and without skeleton files                   # FeatureContext::userHasBeenCreatedWithDefaultAttributesAndWithoutSkeletonFiles()
>     And user "Alice" has uploaded file with content "ownCloud test text file 0" to "/textfile0.txt"        # FeatureContext::userHasUploadedAFileWithContentTo()
>     And user "Brian" has been created with default attributes and without skeleton files                   # FeatureContext::userHasBeenCreatedWithDefaultAttributesAndWithoutSkeletonFiles()
>     When user "Alice" shares file "textfile0.txt" with user "Brian" using the sharing API                  # FeatureContext::userSharesFileWithUserUsingTheSharingApi()
>     And user "Brian" accepts share "/textfile0.txt" offered by user "Alice" using the sharing API          # FeatureContext::userReactsToShareOfferedBy()
>     Then the HTTP status code should be "200"                                                              # FeatureContext::thenTheHTTPStatusCodeShouldBe()
>     And the content of file "/Shares/textfile0.txt" for user "Brian" should be "ownCloud test text file 0" # FeatureContext::contentOfFileForUserShouldBe()
> 
>     Examples:
>       | ocs_api_version |
>       | 1               |
>         Sharing::userReactsToShareOfferedBy could not find share /textfile0.txt, offered by Alice to Brian
>         Failed asserting that null is not null.
>       | 2               |
>         Sharing::userReactsToShareOfferedBy could not find share /textfile0.txt, offered by Alice to Brian
>         Failed asserting that null is not null.
> ```
> 
> How is that related to the changes in this PR???
> 
> The PR needs a rebase anyway. Maybe that will help.

After rebasing and removing the two expected failures CI now passes. Thanks for the pointers!

File: changelog/unreleased/update-web-1.0.1.md [hunk 2]
```diff
@@ -0,0 +1,8 @@
+Change: Update ownCloud Web to v1.0.1
+
+Tags: web
+
+We updated ownCloud Web to v1.0.1. Please refer to the changelog (linked) for details on the web release.
+
+https://github.com/owncloud/ocis/pull/1191
+https://github.com/owncloud/phoenix/releases/tag/v1.0.1
```
Comment (phil-davis) [inline changelog/unreleased/update-web-1.0.1.md:8]:
```suggestion
https://github.com/owncloud/web/releases/tag/v1.0.1
```

How did this end up pointing to the old repo?
The link redirects to `owncloud/web` so it is not broken.

**Review by phil-davis — APPROVED** [review]
Maybe you want to adjust link in the changelog. But actually it works, so maybe not worth having to let CI run again.

Edit: I committed the changelog change - may as well get it fixed.

**Comment by phil-davis** [comment]
Note: this also fixes QA isssue #1197

**Comment by kulmann** [comment]
> Maybe you want to adjust link in the changelog. But actually it works, so maybe not worth having to let CI run again.
> 
> Edit: I committed the changelog change - may as well get it fixed.

Thanks for taking care!

**Pull request merged**